controllers/result_test.go

```diff
@@ -185,6 +185,114 @@ var _ = Describe("ResultController", func() {
 		})
 	})
 
+	Describe("GET /results/:id", func() {
+		Context("given the user already signed in", func() {
+			Context("given a valid result id", func() {
+				It("renders with status 200", func() {
+					user := FabricateUser(faker.Email(), faker.Password())
+					result := FabricateResultWithParams(user, "some specific keyword", models.ResultStatusCompleted)
+
+					response := MakeAuthenticatedRequest("GET", fmt.Sprintf("/results/%d", result.Id), nil, nil, user)
+
+					Expect(response.StatusCode).To(Equal(http.StatusOK))
+				})
+
+				It("display result information", func() {
+					user := FabricateUser(faker.Email(), faker.Password())
+					result := FabricateResultWithParams(user, "some specific keyword", models.ResultStatusCompleted)
+					adLink1 := FabricateAdLink(result)
+					adLink2 := FabricateAdLink(result)
+					link := FabricateLink(result)
+
+					response := MakeAuthenticatedRequest("GET", fmt.Sprintf("/results/%d", result.Id), nil, nil, user)
+					responseBody := GetResponseBody(response)
+					Expect(responseBody).To(ContainSubstring("some specific keyword"))
```

Comment:
Shall we add a space before this line so that it separates with request/response block?

Reply:
added in [a256e8b](https://github.com/carryall/go-google-scraper-challenge/pull/72/commits/a256e8b705397f089a17f323e185c4145726b5eb)